app/models/user.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class User extends AppModel
+{
+    public function getUserDetail($id)
+    {
+        $db = DB::conn();
+        $row = $db->row('SELECT * FROM user WHERE id = ?', array($id));
+        return $row;
```

Comment:
about the return value being an object? what if I actually expected it to return an array? Is it a bad practice to return array?
throw exception done

Thanks!